build.gradle

```diff
@@ -286,4 +286,4 @@ retrolambda {
   javaVersion JavaVersion.VERSION_1_7
   incremental true
     jvmArgs '-noverify'
-}
\ No newline at end of file
+}
```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR.